### PR TITLE
fix issue #4131

### PR DESCRIPTION
**Pull request by AndreyOrlov**

### CODE REVIEW

**Comment by pjcozzi** [comment]
Thanks for the fix, @AndreyOrlov.  Can you update [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md)?

Also, sorry for the slow response; most of the team was at SIGGRAPH last week.

@tfili can you review and merge this?  Is it possible to write a reasonable unit test?

For #4131.

**Comment by pjcozzi** [comment]
Everyone - we have a CLA.

**Comment by tfili** [comment]
The change looks good. @AndreyOrlov could you merge in master. Also, can you add a unit test. Having a tesselated `LineString` with a `LineStyle` that has a `color` should do the trick. Then just check that there is a corridor and that the material property is correct.

Thanks

File: Source/DataSources/KmlDataSource.js [hunk 1]
```diff
@@ -1150,7 +1150,7 @@ define([
                 entity.corridor = corridor;
                 corridor.positions = coordinates;
                 if (defined(polyline)) {
-                    corridor.material = defined(polyline.material) ? polyline.material.color : Color.WHITE;
+                    corridor.material = defined(polyline.material) ? polyline.material.color.getValue() : Color.WHITE;
```
Comment (mramato) [inline Source/DataSources/KmlDataSource.js]: `getValue` requires `time` as its first parameter, if we are 100% sure that `.material.color` will be a constant value (maybe it's impossible to make it time varying with kml?) then we should still use `Iso860.MINIMUM_VALUE` here.  @tfili can you check if this is indeed always constant.

Comment (tfili) [inline Source/DataSources/KmlDataSource.js]: Yes, colors in KML are always constant (excluding the `highlight` style in `StyleMap`s, which isn't supported in Cesium). I would just change it to use `Iso860.MINIMUM_VALUE` as @mramato suggested.

**Comment by AndreyOrlov** [comment]
@tfili, @mramato I added unit test and `Iso860.MINIMUM_VALUE`.

**Comment by mramato** [comment]
@tfili feel free to merge if you are happy.

**Comment by tfili** [comment]
@AndreyOrlov Looks good. Need you to merge in master again. There appear to be another set of conflicts. I'll merge once it's updated.

**Comment by AndreyOrlov** [comment]
@tfili Done.

**Comment by tfili** [comment]
Thanks @AndreyOrlov

**Pull request merged**